scripts/projectUpdater.ps1

```diff
@@ -39,6 +45,26 @@ function _openMergeWindow ($_path1, $_path2) {
 _checkArg $projectFolder
 _checkArg $projectName
 
+# the accept all is optional
+if ([string]::IsNullOrEmpty($acceptAll)) {
+    $acceptAll = $false
+} else {
+    if ($acceptAll -eq "1") {
+        # ask for confirmation
+        Write-Host -ForegroundColor Yellow "You are about to accept all changes"
```

Comment:
"You are about to accept all incoming changes from the updated template"

Reply:
check last commit